tensorflow/lite/micro/python/interpreter/src/tflm_runtime.py

```diff
@@ -95,6 +95,17 @@ def from_bytes(self,
     return Interpreter(model_data, custom_op_registerers, arena_size,
                        num_resource_variables)
 
+  def print_allocations(self):
+    """Invoke the RecordingMicroAllocator to print the arena usage.
+
+    This should be called after `invoke()`.
+
+    Returns:
+      This method does not return anything, but It dumps the arena
+      usage to stderr.
+    """
+    return self._interpreter.PrintAllocations()
```

Comment:
If this doesn't return anything, we shouldn't have `return` here, right?

Reply:
Correct. Let me remove that.